lib/nectar/worker.ex

```diff
@@ -125,29 +151,263 @@ defmodule Nectar.Worker do
   end
 
   defp write_response({:error, reason}, client),
-    do: build_response(500, "Internal Server Error", inspect(reason)) |> send_response(client)
+    do: write_response(%Response{status_code: 500, body: inspect(reason)}, client)
+
+  defp write_response(%Request{} = request, client),
+    do:
+      write_response(
+        %Response{version: request.version, status_code: 200, body: "Hello, world!"},
+        client
+      )
+
+  ##
+  ## 1xx - informational responses
+  ##
+
+  defp write_response(%Response{status_code: 100, status_message: nil} = response, client),
```

Comment:
This takes a lot of lines. Maybe looking it up from a map would be just as good? And we could put it in the `request.ex` file?

I don't think it should hold up the PR but this file is getting big.

Reply:
That can probably be condensed down to a macro, TBH. If it's going to move anywhere, it should be to `response.ex` instead of `request.ex`.

Reply:
Heh, yeah I meant `response.ex` not awake yet I guess :)